src/os/state/v4/baselayerstate.js

```diff
@@ -492,6 +504,22 @@ os.state.v4.BaseLayerState.prototype.configKeyToXML = function(layerConfig, type
         os.xml.appendElement(os.state.v4.LayerTag.ALPHA, layerEl, value);
       }
       break;
+    case 'fillOpacity':
```

Comment:
This should be added to `os.style.StyleField`.

Reply:
I switched it to use the reference defined in os.style.StyleField